src/swish/http.ss

```diff
@@ -211,14 +211,16 @@
                              (conn:start&link cache-mgr ip op options)])
                            (http:handle-input conn url-handler options)))))))])
           (monitor pid)
-          `#(no-reply ,($state copy* [pid->op (ht:set pid->op pid op)])))]
+          `#(no-reply ,($state copy* [pid->op (ht:set pid->op pid (cons ip op))])))]
```

Comment:
Should the name `pid->op` change to `pid->ports`?

Reply:
Good thought!